integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -682,7 +684,7 @@ public void ttlExpiredCreateFileTest() throws Exception {
         mFsMaster.getFileInfo(mFsMaster.getFileId(new TachyonURI("/testFolder/testFile1")));
     Assert.assertEquals(fileId, folderInfo.fileId);
     Assert.assertEquals(ttl, folderInfo.ttl);
-    CommonUtils.sleepMs(5000);
+    HeartbeatScheduler.await(mThreadName,5000, TimeUnit.MILLISECONDS);
```

Comment:
This is not a correct use of the heartbeat scheduler. Instead, you should use schedule execution of the TTL checker to avoid the 5 second wait. Take a look at `HeartbeatThreadTest` for an example of a correct use.